benchmark/strong_scaling_distribution_rate.cpp

```
@@ -0,0 +1,302 @@
+      std::cout << options.help() << std::endl;
+    }
+    MPI_Finalize();
+    return 0;
+  }
+
+  BenchmarkOptions opts;
+  opts.expected_us = args["expected_us"].as<uint64_t>();
+  opts.duration_s = args["duration_s"].as<double>();
+  opts.bundle_target_ms = args["bundle_target_ms"].as<uint64_t>();
+  opts.remove_root_from_distribution = args.count("rm_root") > 0;
+  opts.distributor = parse_distributor(args["distribution"].as<std::string>());
```

Cppcheck (2.19.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)